### DISCUSSION TITLE
Intellisense does not put the variants first.

### DISCUSSION THREAD

**Discussion by Blankeos** [heroui-inc/tailwind-variants#102] [Q&A]
So I'm using Svelte-Kit. I think this is cause by the Tailwind Intellisense sort of overriding the intellisense when inside a class="" attribute, but it still feels unintuitive to not see what my variants are right away.

These are my variants:
```ts
import { tv } from 'tailwind-variants';

export const button = tv({
  base: 'rounded-full font-circularstd shadow',
  variants: {
    color: {
      primary: 'bg-primary-500 text-white ',
      secondary: 'bg-white text-primary-500'
    },
    size: {
      lg: 'px-20 py-4 text-2xl'
    }
  },
  defaultVariants: {
    size: 'lg',
    color: 'primary'
  }
});
```

This is how it looks when I use it:

![image](https://github.com/nextui-org/tailwind-variants/assets/38070918/dadc2d71-ad87-4f26-82c5-c8b4f9e487a2)

Here are some relevant `settings.json` configurations:
```json
  "tailwindCSS.includeLanguages": {
    "javascript": "javascript",
    "typescript": "typescript",
    "plaintext": "javascript",
    "Svelte": "html"
  },
  "tailwindCSS.emmetCompletions": true,
  "tailwindCSS.experimental.classRegex": [
    // Tailwind-Variants
    ["tv\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
  ],
  "tailwindCSS.experimental.configFile": null,
  "tailwindCSS.classAttributes": ["class", "className", "ngClass"],
```

**Comment by mskelton** [comment]
Closing per the comment here: https://github.com/nextui-org/tailwind-variants/issues/25#issuecomment-1783669714
